### PR TITLE
make constitutional amendments proposed in rt#7790

### PR DESCRIPTION
**Pull request by cg505**
These changes clarify procedures surrounding special elections.

### CODE REVIEW

File: ocfweb/docs/docs/docs/operatingrules/constitution.md [hunk 1]
```diff
@@ -168,21 +173,31 @@ If an elected office becomes vacant, a special election shall be held at the
 next Board meeting to elect a replacement, unless a regular election is being
 held at the next Board meeting.
 
+##### 5.2.4 Nominations {amended-2019-02-11-5-2-4}
+
+Nominations for elected offices can take place either at the Board meeting
+before the election, or two Board meetings before the election. Nominations
+for a special election may also be made electronically over a medium
+available to all Directors up to 48 hours before the special election or 48
+hours after the vacancy begins, whichever is later. Nominations can only
```
[the remark this answers is not in the document]
Comment (cg505) [reply ocfweb/docs/docs/docs/operatingrules/constitution.md]: Suppose the vacancy begins 48 hours before the next board meeting. If it just said "up to 48 hours before the special election" then there would be no possible period for nominations.

File: ocfweb/docs/docs/docs/operatingrules/constitution.md [hunk 2]
```diff
@@ -168,21 +173,31 @@ If an elected office becomes vacant, a special election shall be held at the
...
+come from Board members, and every nomination requires the approval of one
```
[the remark this answers is not in the document]
Comment (cg505) [reply ocfweb/docs/docs/docs/operatingrules/constitution.md]: This sentence is not new, merely moved. I think we should avoid making any changes beyond those necessary. Additionally I feel like it makes sense to require seconds on nominations.

File: ocfweb/docs/docs/docs/operatingrules/constitution.md [hunk 3]
```diff
@@ -168,21 +173,31 @@ If an elected office becomes vacant, a special election shall be held at the
...
+other Board member who is not on the ticket and the approval of the
+nominated ticket to be placed on the ballot. Self-nominations are allowed.
```
Comment (jameslzhu) [inline ocfweb/docs/docs/docs/operatingrules/constitution.md]: s/nominated ticket/nominee?

**Comment by cg505** [comment]
The amendment was passed in its current for by BoD today.

**Pull request merged**